### ISSUE TITLE
node-forge (HMAC hashing library) won't work with React Native's strict mode

### ISSUE THREAD

**Issue by nigelnindo** [Ma3Route/node-sdk#12]
Looking into the source code for node-forge, it deletes local variable (Line 53, forge.js) making it incompatible with React Native's strict mode. Consider looking into this, or use a different hashing library.

**Comment by GochoMugo** [comment]
Which version of **node-forge** are you using?

You could provide us with the output of:

```bash
$ npm ls node-forge
```

**Comment by nigelnindo** [comment]
Version **0.6.42**. Quick check on npm shows that's the latest version. **node-sdk** uses **0.6.34**, but had tried to update it, and still got same error.

**Comment by GochoMugo** [comment]
This issue is related to https://github.com/digitalbazaar/forge/issues/333. Please navigate there to find more information on possible work-arounds until the library fixes the bug.

Otherwise, this issue introduces the question of support of this library in **browser** environments. It was built for Node.js runtime in mind. We might look into this aspect.

**Comment by GochoMugo** [comment]
I have reviewed your PR at #13 and I highly appreciate your effort.

But it brings up the following concerns:

* I am reluctant on swapping libraries quite easily. This is particularly discouraged if it is due to bug fixes, that can be handled.
* We are **not** yet designed to handle browser environments correctly

However, I have drafted a new PR i.e. #14, that uses the built-in crypto library. This removes the dependency for a crypto library, which is a positive side-effect for us. Would such a fix suffice your scenario?
